### ISSUE TITLE
Bug report mechanism

### ISSUE THREAD

**Issue by advayDev1** [brunobowden/j2objc-gradle#69]
When a failure occurs in any step, we should dump some standard info to the error logs:
  - version of j2objc you are running
  - version of clang toolchain
  - location of j2objc home
  - version of j2objc-gradle (if we do release tags)

Then we can have users paste that info into any bug report.

**Comment by brunobowden** [comment]
Great idea. Bonus points if it allows opt-in reporting to the server...
I'll ask the Gradle guys if they're doing anything to support that.

**Comment by advayDev1** [comment]
Yes you mentioned that on another PR, I'm privacy-risk-averse and so suggested this non-magical alternative. =)

**Comment by brunobowden** [comment]
Agreed for now
